draft-ietf-quic-multipath.md

```diff
@@ -587,26 +579,25 @@ the ACK frame as well as packet protection as described in the following subsect
 The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
 acknowledge 1-RTT packets.
 Compared to the QUIC version 1 ACK frame, the ACK_MP frame additionally
-contains a Path ID.
-The Path ID is used to distinguish packet number spaces for different
-paths and is simply derived from the sequence number of
-Destination Connection ID.
-Therefore, the Path ID for 1-RTT packets can be identified
-based on the Destination Connection ID in each packet.
+contains the Receiver's Destination Connection ID Sequence Number field
+to distinguish the Connection ID-specific packet number space
+acknowledged.
 
 Acknowledgements of Initial and Handshake packets MUST be carried using
 ACK frames, as specified in {{QUIC-TRANSPORT}}. The ACK frames, as defined
-in {{QUIC-TRANSPORT}}, do not carry path identifiers. If for any reason
-ACK frames are received in 1-RTT packets while the state of multipath
-negotiation is ambiguous, they MUST be interpreted as acknowledging
-packets sent on path 0.
+in {{QUIC-TRANSPORT}}, do not carry the Receiver's Destination
+Connection ID Sequence Number field. If multipath has been successfully
+negotiated, ACK frames in 1-RTT packets acknowledge packets sent with
+the Connection ID having sequence number 0.
```

Comment:
```suggestion
Connection ID Sequence Number field. If multipath has been successfully
negotiated, ACK frames in 1-RTT packets MUST be interpreted as acknowledging
packets on the path they the packet with the ACK frames been received.
```

Reply:
No, please do not apply this change. This PR is editorial in nature, and should not change the consensus so far, i.e., the old ACK frame acknowledges CID number 0. Quentin's original text is fine.

Reply:
Yes, this changes goes maybe a bit far for this PR but Quentin's text is not right because he removed the "If for any reason  ACK frames are received in 1-RTT packets while the state of multipath negotiation is ambiguous" part. In this ambiguous state all acks are mapped to the seq number 0. However, this is not true later. My proposal is not to change the consensus but rather say the more general thing.

Reply:
Yes, let's avoid introducing design changes here. Also, I think relying on the reception 4-tuple may lead to ambiguous state at the receiver.

Reply:
For the ambiguous part, I'm not sure to understand when the negotiation of multi path could be ambiguous. But so far the less ambiguous approach (as I understood so far) is that ACK frames (always) relate to CID number 0.

Reply:
I was assuming an ACK frame always corresponds to the path it was sent on. However, it seem this is simply not defined in the draft and we probably need to open an issue to clarify this.

Reply:
No, we specifically do not want implementations to continue sending ACK frames after negotiating multipath. The notion of "current path" is ambiguous -- for example, it does change in case of NAT rebinding. Adding the CID-sequence-number in the ACK_MP guarantee that there is no such ambiguity. As for nake ACK, the equivalence to "cid-number=0" is straightforward.